plugins/src/main/java/opengrok/auth/plugin/LdapUserPlugin.java

```diff
@@ -157,8 +158,7 @@ public void fillSession(HttpServletRequest req, User user) {
         String dn = null;
```

Comment:
the user is still set to `null` (if that is something you tried to avoid based on your description)

Reply:
line 151